pkg/windows/install_salt.ps1

```diff
@@ -13,6 +13,27 @@ ready to be packaged.
 install_salt.ps1
 
 #>
+param(
+    [Parameter(Mandatory=$false)]
+    [Alias("b")]
+    # Don't pretify the output of the Write-Result
+    [String] $BuildDir,
+
+    [Parameter(Mandatory=$false)]
+    [Alias("c")]
+    # Don't pretify the output of the Write-Result
+    [Switch] $CICD,
+
+    [Parameter(Mandatory=$false)]
+    # Don't install. It should already be installed
+    [Switch] $SkipInstall,
+
+    [Parameter(Mandatory=$false)]
+    [String] $SourceTarball,
```

Comment:
The comments in between these parameters are the docs. If you run the script with `-Help` it will display the commented text to descript what the switch does. These new parameters need docs here, like the ones above.

Reply:
Oh! Nice! Will add.